app/frontend/src/features/search/SearchPage.tsx

```diff
@@ -15,14 +15,18 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     alignContent: "stretch",
     flexDirection: "column-reverse",
-    height: "100%",
+    position: "fixed",
+    top: "4rem",
```

Comment:
this should be `theme.shape.navPaddingXs` and then `theme.shape.navPaddingSmUp` at the right breakpoint.